oneanddone/users/views.py

```diff
@@ -79,10 +79,14 @@ class ProfileDetailsView(generic.DetailView):
     template_name = 'users/profile/detail.html'
 
     def get_context_data(self, **kwargs):
+        context = super(ProfileDetailsView, self).get_context_data(**kwargs)
         all_attempts_finished = self.object.user.taskattempt_set.filter(state=TaskAttempt.FINISHED)
         paginator = Paginator(all_attempts_finished, 20)
         page = self.request.GET.get('page', 1)
-
+        user_profile = self.object
+        if user_profile.bugzilla_email:
+            context['activity_url'] = 'https://bugzilla.mozilla.org/page.cgi?'\
```

Comment:
Actually, @tessie, upon reviewing this again, and thinking about unit tests, I realized that this really should be in the model and not in the view. Rather than calculating this value in the view and adding it to the context, we should have a property in the `UserProfile` model that returns the correct url. Then the template can just use that model property.

Does that make sense?

Could you make that change, and also add a unit test for that new model property?